buildpipeline/pipelinejobs.groovy

```diff
@@ -28,10 +33,19 @@ def windowsPipeline = Pipeline.createPipelineForGithub(this, project, branch, 'b
 ['netcoreapp'].each { targetGroup ->
 	['Debug', 'Release'].each { configurationGroup ->
 		['Windows x64'].each { osName ->
-            def parameters = ['Config':configurationGroup, 'OuterLoop':false]
-            windowsPipeline.triggerPipelineOnGithubPRComment("${osName} ${configurationGroup} Build", "(?i).*test\\W+portable\\W+windows\\W+${configurationGroup}\\W+pipeline.*", parameters)
+            // One for just innerloop
+            windowsPipeline.triggerPipelineOnGithubPRComment("Portable ${osName} ${configurationGroup} Build", "(?i).*test\\W+portable\\W+windows\\W+${configurationGroup}\\W+pipeline.*",
+                ['Config':configurationGroup, 'OuterLoop':false])
+            // Add one for outerloop
+            windowsPipeline.triggerPipelineOnGithubPRComment("Portable Outerloop ${osName} ${configurationGroup} Build", "(?i).*test\\W+outerloop\\W+portable\\W+windows\\W+${configurationGroup}\\W+pipeline.*",
```

Comment:
Is there room anywhere to include a description of where things will actually run?  e.g. to tell the caller than this one leg will result in runs on Win7, Win8, Win10, and Nano?